src/mscorlib/src/System/Reflection/Module.cs

```diff
@@ -1155,7 +1155,7 @@ public override FieldInfo GetField(String name, BindingFlags bindingAttr)
         public override MethodInfo[] GetMethods(BindingFlags bindingFlags)
         {
             if (RuntimeType == null)
-                return new MethodInfo[0];
+                return Array.Empty<MethodInfo>();
```

Comment:
These ```RuntimeType == null``` cases should be rare (if ever?), these arrays are being returned publically and so there's non-zero compat risk, etc.  Doesn't seem worth further investigation.  Let's revert these.